schema_based_on_diagram.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE medical_database;
+
+-- TABLE patients
+CREATE TABLE patients (id BIGSERIAL PRIMARY KEY, name VARCHAR(120), date_of_birth DATE);
+
+-- TABLE medical histories
+CREATE TABLE medical_histories (id BIGSERIAL PRIMARY KEY, admitted_at TIMESTAMP, patient_id INT REFERENCES patients(id), status VARCHAR(120));
+
+-- TABLE invoices
+CREATE TABLE invoices (id BIGSERIAL PRIMARY KEY, total_amount INT, generated_at TIMESTAMP, payed_at TIMESTAMP, medical_history_id INT REFERENCES medical_histories(id));
+
+-- TABLE invoice_items
+CREATE TABLE invoice_items (id BIGSERIAL PRIMARY KEY, unit_price INT, quantity INT, total_price INT, invoice_id INT REFERENCES invoices(id), treatment_id INT);
```

Comment:
Great job team 🤝

- [ ] Kindly make sure the data types of the `quantity`, `unit-price`, and `total_amount` are decimal instead of integers both in the `invoices`, and the `invoice_items` tables.

Reply:
DONE!